contract/models/contract_line.py

```diff
@@ -35,10 +35,6 @@ class ContractLine(models.Model):
         string="Analytic account",
         comodel_name="account.analytic.account",
     )
-    analytic_tag_ids = fields.Many2many(
```

Comment:
If you remove this, you should inherit from analytic.mixin instead

Reply:
Ok - will inherit from analytic.mixin

Reply:
done